src/kakarot/instructions/system_operations.cairo

```diff
@@ -452,6 +452,7 @@ namespace CallHelper {
         // During teardown we update the memory using this offset
         let return_data: felt* = alloc();
         assert [return_data] = ret_offset;
+        Helpers.fill(ret_size, return_data+1, 0);
```

Comment:
you don't format automatically in your workflow

Reply:
since we append data to `return_data` in the `exec_return`, how do this work because the segment is already written?